apps/admin/src/adminResourceOptions/coursesOptions.ts

```diff
@@ -64,6 +66,7 @@ export const coursesConfigOptions: ResourceOptions = {
     "status",
     "file",
     "lessons",
+    "currency",
```

Comment:
I think it would be nice to move it under the price instead of keeping this at the end of the list